rider-core/pom.xml

```diff
@@ -34,6 +34,21 @@
             <artifactId>jackson-databind</artifactId>
         </dependency>
 
+        <!-- https://mvnrepository.com/artifact/it.unibo.tuprolog/solve-classic-jvm -->
+        <dependency>
+            <groupId>it.unibo.tuprolog</groupId>
+            <artifactId>solve-classic-jvm</artifactId>
+            <version>0.20.2</version>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/it.unibo.tuprolog/parser-theory-jvm -->
+        <dependency>
+            <groupId>it.unibo.tuprolog</groupId>
+            <artifactId>parser-theory-jvm</artifactId>
+            <version>0.20.2</version>
+        </dependency>
```

Comment:
I checked the dependency tree and it's bringing `kotlin-stdlib`, anyone from @database-rider/rider-community knows if it could be a compatibility problem for users trying to upgrade to this rider version? (Not talking about java8 which is about time, thanks @coiouhkc!)

In terms of size in the final jar it is negligible so it is fine by me, just concerned  about compat issues.